apps/sms/index.html

```diff
@@ -608,13 +608,15 @@ <h1 class="message-subject">
 
     <div id="messages-group-header-tmpl" class="hide">
       <!--
-      <bdi>${name}</bdi><bdi dir="ltr"> (+${participantCount})</bdi>
+      <span class="group-header-title">
+        <bdi class="ellipsis-dir-fix">${name}</bdi><bdi dir="ltr"> (+${participantCount})</bdi>
+      </span>
       -->
     </div>
 
     <div id="messages-header-tmpl" class="hide">
       <!--
-      <bdi>${name}</bdi>
+      <bdi class="ellipsis-dir-fix">${name}</bdi>
```

Comment:
With this style I see that some header content is "less centered" (presumably because of -moz-fit-content):

1. With the patch: https://people.mozilla.org/~azasypkin/bug-1138340/issue.png
2. Master: https://people.mozilla.org/~azasypkin/bug-1138340/master.png

Do you think it's our issue or gaia-header or maybe not issue at all? I'm not sure how this gaia-heder\font-fit centering stuff works and Julien and Wilson both on PTO :)